stdlib/email/headerregistry.pyi

```diff
@@ -90,12 +115,16 @@ class ContentTypeHeader(ParameterizedMIMEHeader):
     def value_parser(value: str) -> ContentType: ...
 
 class ContentDispositionHeader(ParameterizedMIMEHeader):
+    max_count: ClassVar[int | None]
+    # init is redefined but has the same signature as parent class, so is omitted from the stub
     @property
     def content_disposition(self) -> str: ...
     @staticmethod
     def value_parser(value: str) -> ContentDisposition: ...
 
 class ContentTransferEncodingHeader:
+    max_count: ClassVar[int | None]
+    def init(self, name: str, *, parse_tree: TokenList, defects: Iterable[MessageDefect]) -> None: ...
```

Comment:
Why is this redefined? The class passes all arguments to `super().init()` as is.

Reply:
Yes, but this class isn't subclassing anything, so it's not actually "redefining" `init`